src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysMaxSize = 3;
+        int resultIndex = 2;
+        String[] type = new String[arraysMaxSize];
+        int[] amount = new int[arraysMaxSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] values = value.split(importantSymbol);
+                for (int i = 0; i < arraysMaxSize; i++) {
+                    if (type[i] == null && values[operationTypeIndex] != null) {
+                        type[i] = values[operationTypeIndex];
+                        amount[i] += Integer.parseInt(values[amountIndex]);
+                        break;
+                    }
+                    if (type[i] != null && type[i].equals(values[operationTypeIndex])) {
+                        amount[i] += Integer.parseInt(values[amountIndex]);
+                        break;
+                    }
+                }
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
```

Comment:
The catch block correctly rethrows the exception as a `RuntimeException`, which is good practice. However, the error message could be more informative by including the file name that caused the error: `"Can't read data from the file " + fromFileName`.